spec/factories/courses.rb

```diff
@@ -50,6 +50,10 @@
       level { :secondary }
     end
 
+    trait :engineers_teach_physics do
+      campaign_name { "engineers_teach_physics" }
+    end
+
```

Comment:
```suggestion
```

^^ That looks way better

Reply:
https://media.giphy.com/media/SwyTq2jJxc9im6BYnN/giphy.gif